src/lib/token-utils.ts

```diff
@@ -40,7 +40,7 @@ export async function getTokenInfo(
       totalSupply: values[3],
     };
   } catch (error) {
-    console.error('Error getting token info from contract');
+    console.error('Error getting token info from contract', error);
```

Comment:
Not sure what convention is, but let's remove the actual error, no need for the client to see the detailed info as far as I'm concerned.